src/main/resources/au/com/centrumsystems/hudson/plugin/buildpipeline/BuildPipelineView/bpp.jelly

```diff
@@ -365,19 +367,23 @@
 									<!-- Only adding the revision-cell once. -->
 									<j:if test="${x == 0}"><td class="revision-cell" rowspan="${buildGrid.rows}">
 										<table class="revision rounded build-card">
-											<tbody>
-												<tr class="header">
```

Comment:
@oleg-nenashev The bad practice is not to use custom CSS overriding the default implementation but to use not semantic pseudo classes, etc...

As I said in other comments, we have to do an effort to improve this part of Jenkins project. It spend much time about binary compatibility, API compatibility, etc... but the UI is a very important part of the code.